### PR TITLE
Test "read-your-writes" in view change tests

### PR DESCRIPTION
**Pull request by Liorgal28**
1) Refactor "wait_for_view_change" into "wait_for_view".

2)Insert "read your writes" functionality after view change.

### CODE REVIEW

File: tests/test_skvbc_auto_view_change.py [hunk 1]
```diff
@@ -104,12 +106,35 @@ class SkvbcAutoViewChangeTest(unittest.TestCase):
...
+        self.assertTrue(reply.success)
+        self.assertEqual(last_block + 1, reply.last_block_id)
+
+        last_block = reply.last_block_id
+
+        # Read the last write and check if equal
+        # Get the kvpairs in the last written block
+        data = await client.read(skvbc.get_block_data_req(last_block))
+        kv2 = skvbc.parse_reply(data)
+        self.assertDictEqual(kv2, dict(kv))
```
Comment (teoparvanov) [inline tests/test_skvbc_auto_view_change.py]: Can you move this to skvbc.py (as a member of the SimpleKVBCProtocol class), and reuse it from there?

File: private_replica_0 [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+num_replicas: 4
+num_ro_replicas: 0
+f_val: 1
+c_val: 0
+replica_id: 0
+read-only: 0
+
+# RSA non-threshold replica public keys
+rsa_public_keys:
+  - 30820120300D06092A864886F70D01010105000382010D00308201080282010100D2837718AE65367B664C3011B4D9E70E9EEE1E91E8DFC4D6EFBE1C6C6FEA260FEE0821F3E29B83A57E91AA64FFFD5152F0469996FE7C60F4778FF9FAE8A5C280F9BB59004FAF321790D4177DCC1976560558E38B89CF21B80DD37BBE87813CA030BA600D85AD9F0DB5F15AC0B3CB4E2DBDCD200CA99630FEC25441F5EDFFE2674A907EFB3E28F45418841B9DC375563D2C1FE7D6F9BD0C1EB40CF464CB7F37C01E3D1815E5954542BF3FEC534D5FD69E63BF4B21B75988B91CDD49DCC6EB3955559A3956427B38AF59D16853F5F8319973BCD9602DA820C6729C4FBD0E1E40A23246AEE644A0E0C68E330850B63D4F96F36E81A4B1C178B3CA6558800438ED3F020111
```
Comment (teoparvanov) [inline private_replica_0]: this should not be part of the PR

File: tests/util/dev/pythonSudo.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 # This shell script allows running the Python interpreter as root
 # NB: the user needs sudo NOPASSWD enabled
-sudo PYTHONPATH=$PYTHONPATH python3 "$@"
+sudo PYTHONPATH=$PYTHONPATH python3 "$@"
```
Comment (teoparvanov) [inline tests/util/dev/pythonSudo.sh]: please revert this change

File: tests/venv2/bin/activate [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# This file must be used with "source bin/activate" *from bash*
+# you cannot run it directly
+
+deactivate () {
+    # reset old environment variables
+    if [ -n "${_OLD_VIRTUAL_PATH:-}" ] ; then
+        PATH="${_OLD_VIRTUAL_PATH:-}"
+        export PATH
+        unset _OLD_VIRTUAL_PATH
+    fi
```
Comment (teoparvanov) [inline tests/venv2/bin/activate]: please do not submit virtual environment code here

File: tests/test_skvbc_auto_view_change.py [hunk 5]
```diff
@@ -51,15 +51,16 @@ class SkvbcAutoViewChangeTest(unittest.TestCase):
...
         # do nothing - just wait for an automatic view change
-        await bft_network.wait_for_view_change(
-            replica_id=random.choice(
-                bft_network.all_replicas(without={initial_primary})),
-            expected=lambda v: v > initial_primary,
-            err_msg="Make sure automatic view change has occurred."
-        )
+        await bft_network.wait_for_view(replica_id=random.choice(
+            bft_network.all_replicas(without={initial_primary})), expected=lambda v: v > initial_primary,
+            err_msg="Make sure automatic view change has occurred.")
```
Comment (teoparvanov) [inline tests/test_skvbc_auto_view_change.py]: Can you preserve the existing code formatting, and only change the function name?

File: tests/test_skvbc_auto_view_change.py [hunk 6]
```diff
@@ -77,13 +78,14 @@ class SkvbcAutoViewChangeTest(unittest.TestCase):
...
```
Comment (teoparvanov) [inline tests/test_skvbc_auto_view_change.py]: again, why are you changing the formatting here? I'd only expect to see the function name change

File: tests/test_skvbc_auto_view_change.py (hunk 5 above)
Comment (Liorgal28) [reply tests/test_skvbc_auto_view_change.py]: i have changed only the name, the format is changed because the function name has been shorter
